kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -192,6 +192,22 @@ deploy_kasfleetmanager() {
   "supported_instance_type": "standard,eval"
   "status": "ready"
   "cluster_dns": "'${DATA_PLANE_CLUSTER_DNS_NAME}'"
+' \
+    -p SUPPORTED_INSTANCE_TYPES='
+- "id": "standard"
+  "display_name": "Standard"
+  "sizes":
+  - "id": "x1"
+    "ingressThroughputPerSec": '"${KAFKA_CAPACITY_INGRESS_THROUGHPUT}"'
+    "egressThroughputPerSec": '"${KAFKA_CAPACITY_EGRESS_THROUGHPUT}"'
+    "totalMaxConnections": '${KAFKA_CAPACITY_TOTAL_MAX_CONNECTIONS}'
+    "maxConnectionAttemptsPerSec": '${KAFKA_CAPACITY_MAX_CONNECTION_ATTEMPTS_PER_SEC}'
+    "maxDataRetentionSize": '"${KAFKA_CAPACITY_MAX_DATA_RETENTION_SIZE}"'
+    "maxDataRetentionPeriod": '"${KAFKA_CAPACITY_MAX_DATA_RETENTION_PERIOD}"'
+    "maxPartitions": '${KAFKA_CAPACITY_MAX_PARTITIONS}'
+    "quotaConsumed": 1
+    "quotaType": "RHOSAK"
+    "capacityConsumed": 1
```

Comment:
Alternatively, you could rely on the default value supplied by KFM: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/templates/service-template.yml#L163

until https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/163 is in which may allow to specify the configuration in some other ways. I feel like the two PRs (#163 and #164) are going to walk on one another in regards to supported instance types.